code/_globalvars/lists/typecache.dm

```diff
@@ -10,3 +10,5 @@ GLOBAL_LIST_INIT(typecache_living, typecacheof(/mob/living))
 GLOBAL_LIST_INIT(typecache_stack, typecacheof(/obj/item/stack))
 
 GLOBAL_LIST_INIT(typecache_machine_or_structure, typecacheof(list(/obj/machinery, /obj/structure)))
+
+GLOBAL_LIST_INIT(typecache_elevated_structures, typecacheof(list(/obj/structure/table,/obj/structure/rack,/obj/machinery/conveyor,/obj/structure/closet))) //This is used for ants.
```

Comment:
1. Auto-doc this
2. Your doc-comment should explain what it *is*, not what something else happens to use it for.

Reply:
What is auto-doc? Other than a transformer character. Is this on github documents?

Reply:
```
/// bla bla bla
var/here
```